src/main/java/io/cryostat/rules/Rule.java

```diff
@@ -140,6 +142,14 @@ public int getMaxSizeBytes() {
         return this.maxSizeBytes;
     }
 
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
```

Comment:
Can these two methods be package-private? It could trip someone up to see `setEnabled` as a visible method here, but calling it doesn't actually have the desired effect, which is really done by `RuleRegistry#enableRule`

Reply:
`RuleRegistry#enableRule` uses the `setEnable`. Do you think that it is fine to set to `protect` instead of `private`?

Reply:
ack, thanks!